field_manager/field_manager.F90

```diff
@@ -829,7 +850,7 @@ end subroutine new_name
 !! needed within various modules. The field manager does not
 !! initialize any of those schemes however. It simply holds the
 !! information and is queried by the appropriate  module.
-subroutine field_manager_init(nfields, table_name)
+subroutine field_manager_init_legacy(nfields, table_name)
```

Comment:
I would prefer `field_manager_init_field_table_original` or something like that.  Legacy is not descriptive enough.

Reply:
renamed to `read_field_table_legacy` and `read_field_table_yaml`